Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20.11.1-slim AS build
+WORKDIR /app
+COPY . ./
+RUN npm install

```

Comment:
RUN npm ci

https://docs.npmjs.com/cli/v10/commands/npm-ci

Reply:
Have added this now